var/spack/repos/builtin/packages/openfoam-org/50-site.patch

```diff
@@ -0,0 +1,42 @@
+diff -uw etc/config.sh/settings.orig etc/config.sh/settings
```

Comment:
Why do we need this patch? Wouldn't overwriting `WM_PROJECT_INST_DIR` in the environment work, too?

Reply:
I am not sure why it is needed. I just tried to apply the same patches that were there already for version `4.1` on this new version of the application. Overwriting would probably work so I can remove the patch altogether if you prefer me to.

Reply:
I simply prefer overwrites over patches as overwrites forward port automatically.

Reply:
Hi @junghans . I have removed the patch altogether. Let me know what you think.

Reply:
There is still some old cruft kicking about there. In previous releases the FOAM_INST_DIR was optionally set by the user prior to sourcing the bashrc and then used internally to avoid the hard-coded WM_PROJECT_INST_DIR. In the meantime, the bashrc is supposed to be smart enough to handle everything and we should actually ditch FOAM_INST_DIR since it will very rarely ever function like its original intent. However, we'd still need some other intermediate variable (foamInstall in some versions) to note the installation directory prior to using foamCleanPath and WM_PROJECT_DIR etc to remove old content.